template/python/cargo_version.py

```diff
@@ -21,176 +19,188 @@
 #   -m PRERELEASE, --prerelease PRERELEASE
 #                         Set pre-prelease string.
 #
-
-import argparse
-import semver
 import toml
+import semver
```

Comment:
Did the linter complain about this? I still remember changing this to be alphabetic because it complained about that a few weeks ago..

Reply:
I used `black` for auto-formatting and then I ran `flake8` and got no errors.

Reply:
Ok. As long as the linters are happy I don't have a strong opinion either way.

Is this in line with the master version of this script in stackable/stackable-utils though?

Reply:
yes